snapshot-labs/sx-monorepo
# feat: add lens and farcaster to user profile

### Summary



Closes: #406

### Features

- Add support for `lens` and `farcaster` social network on user profile

### Fixes

- fix an issue where a new user couldn't be saved without filling all fields
- show an error message when the user ID is not a valid address (before, /profile/hello) was loading an empty user)
- fix an issue where the User page was flashing with "User does not exist" briefly on after update

### How to test

1. Go to your user profile
2. Edit it to fill a lens and farcaster handles
3. It should appear on your profile

### TODO 

- [x] Waiting for merging of https://github.com/snapshot-labs/snapshot.js/pull/1027
